cmd/secret/cmd/root_test.go

```diff
@@ -0,0 +1,118 @@
+package cmd
+
+import (
+	"context"
+	"encoding/json"
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestRoot_Set(t *testing.T) {
+	t.Run("expect one keys", func(t *testing.T) {
+		key := "key value"
+		path := "testFile.txt"
+		version := "test"
+		r := New(version)
```

Comment:
That looks like not very comfortable to pass version when it is not useful (or not making any difference). Take example from https://www.sohamkamani.com/golang/options-pattern/ to refactor `version` into optional argument.